compiler/rustc_mir_transform/src/remove_noop_landing_pads.rs

```diff
@@ -83,9 +83,9 @@ impl RemoveNoopLandingPads {
     fn remove_nop_landing_pads(&self, body: &mut Body<'_>) {
         debug!("body: {:#?}", body);
 
-        // make sure there's a single resume block
```

Comment:
Note that the claim that this ensured that the given block was the only resume block was wrong wrt the previous implementation too.